presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleTester.java

```diff
@@ -39,7 +41,7 @@
     private final TransactionManager transactionManager;
     private final AccessControl accessControl;
 
-    public RuleTester()
+    public RuleTester(Plugin...plugins)
```

Comment:
`RuleTester` is unlikely to benefit from varargs, so  why not `List<Plugin>` here?